cli/tests/upstream.rs

```diff
@@ -2,7 +2,7 @@ mod common;
 
 use {
     common::{Test, TestResult},
-    hyper::{Request, Response, StatusCode},
+    hyper::{header, Request, Response, StatusCode},
```

Comment:
```suggestion
    hyper::{header::HeaderValue, Request, Response, StatusCode},
```

See import comment below :point_down: (_Unsure if this will cause some `cargo fmt` noise_)

Reply:
I need header as well for HOST, btw.

Reply:
`header::{self, HeaderValue}` would work, in that case.

Reply:
Yup, I know. I will change it but isn't this subjective?

Reply:
It is subjective but FWIW there's a pretty strong Rust idiom to prefer imports to qualified paths in the vast majority of cases (unless some kind of disambiguation is needed).

Reply:
Updated.